src/app_utils.py

```diff
@@ -23,6 +24,12 @@ def read_json(json_path):
     return data
 
 
+def read_yaml(yaml_path):
+    assert yaml_path, f'{yaml_path} not exist'
+    with open(yaml_path, 'r') as f:
+        data = yaml.load(f, Loader = yaml.FullLoader)
```

Comment:
Please use `Loader=yaml.SafeLoader` instead.

`FullLoader` raises security concerns and isn't available in `PyYaml > 5.1`.